src/codec/frame/mod.rs

```diff
@@ -116,22 +119,28 @@ pub fn apply_mask_fast32(buf: &mut [u8], mask: [u8; 4]) {
 /// websocket read state
```

Comment:
I prefer to use allow dead code lint here, because these fields are necessary for parsing, user who use this struct should gain full ability without any feature gate.